### PR TITLE
feat: update docs openapi

### PR DESCRIPTION
**Pull request by MiguelMRojas**
- Update OpenApi

### CODE REVIEW

**Review by PedroChaparro — CHANGES_REQUESTED** [review]
Please, add the following to the endpoints that require authentication: 

```yaml
security:
        - bearer: []
```

**Comment by MiguelMRojas** [comment]
> Please, add the following to the endpoints that require authentication:
> 
> ```yaml
> security:
>         - bearer: []
> ```

Done :)

**Review by PedroChaparro — CHANGES_REQUESTED** [review]
1. There are some duplicated endpoints, Eg. POST `/file/upload` and POST `/files`. 
2. The Bearer token should only be added to the endpoints that require authentication. For instance, the `/auth/login` and `/account/register` do not require authentication, while the `/files/download` does.
3. There are some endpoints with incorrect tags. For instance, the `account/register` endpoint should have the `account` tag.

**Comment by MiguelMRojas** [comment]
> 1. There are some duplicated endpoints, Eg. POST `/file/upload` and POST `/files`.
> 2. The Bearer token should only be added to the endpoints that require authentication. For instance, the `/auth/login` and `/account/register` do not require authentication, while the `/files/download` does.
> 3. There are some endpoints with incorrect tags. For instance, the `account/register` endpoint should have the `account` tag.

Done, thanks for comment 💯

File: docs/spec.openapi.yaml [hunk 1]
```diff
@@ -111,32 +83,26 @@ paths:
...
                 $ref: "#/components/schemas/statusResponse"
-  /account/password:
-    patch:
+
+  /account/register:
+    post:
       tags:
-        - Account
-      description: Updates the username password
+        - account
```
Comment (PedroChaparro) [inline docs/spec.openapi.yaml:90]: Should be uppercase: `Account`

File: docs/spec.openapi.yaml [hunk 2]
```diff
@@ -43,64 +43,36 @@ paths:
...
-          description: Internal error
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/statusResponse"
+
   /auth/refresh:
     post:
       tags:
         - Authorization
```
Comment (PedroChaparro) [inline docs/spec.openapi.yaml:50]: Should be `Authentication`

File: docs/spec.openapi.yaml [hunk 3]
```diff
@@ -148,94 +114,75 @@ paths:
               schema:
                 $ref: "#/components/schemas/statusResponse"
 
-  /files:
+  /file/upload:
     post:
       tags:
-        - Files
-      description: Create a new file
+        - File
```
Comment (PedroChaparro) [inline docs/spec.openapi.yaml:120]: Should be `Files`

**Review by PedroChaparro — CHANGES_REQUESTED** [review]

**Review by PedroChaparro — APPROVED** [review]
Note that this spec is not fully compatible with the `.net` one; however, I think all the methods exposed by the gateway are covered by this spec. Let's see what happens.

Squash and merge.

**Pull request merged**